lib/dynamodb.js

```diff
@@ -40,7 +40,6 @@ module.exports = function(configure, connectionConfig = {}) {
 			timeout: 5000,
```

Comment:
@jgrantr with the recent issues with connection timeouts do you think we should up that value fro 2s?

Reply:
from Grant:
make connectionTimeout an env `DYNAMODB_CONNECT_TIMEOUT_MS`  default (5000)
and timeout an env `DYNAMODB_TIMEOUT_MS` default (5000)